cfgov/jinja2/v1/_includes/organisms/data_snapshot.html

```diff
@@ -41,14 +43,13 @@
         {{ value.year_over_year_change_text }}</h3>
     </div>
 
-    {% if value.image %}
```

Comment:
This assumes that `image` is always present yet the help text [above](https://github.com/cfpb/cfgov-refresh/pull/3116/files#diff-bf1cda2d1a0671feb046fdb53fac3546L21) indicates that it is optional. Also the help text still refers to `value.data_month`.